### DISCUSSION TITLE
What's the format of personal access token?

### DISCUSSION THREAD

**Discussion by qinyihao** [lowlighter/metrics#139] [Q&A] [answered]
Hello, I ran into a problem that whatever format I entered for the access token, I still can't build the whole pic.  
![image](https://user-images.githubusercontent.com/40647091/107120903-7fb23500-68ca-11eb-9b07-0d2daed595c5.png)  
If possible, please change it [here](/qinyihao/qinyihao/edit/master/.github/workflows/profile.yml) and open a pull request in that repo. Thanks!

**Comment by lowlighter** [comment]
Hello!

You should revoke your token now as you're not supposed to put them in plain text. Even if you blurred them on the image, anyone accessing your repository can actually see them 😅

It must be used through repository secrets (accessible through your repository settings) like said in this section : [Using GitHub action for metrics](https://github.com/lowlighter/metrics#%EF%B8%8F-using-github-action-on-your-profile-repository-5-min-setup).

There is a collapsible section "How to setup" which can be expanded and it contains a step-by-step tutorial with pictures 🙂 

`committer_token` should litterally be set to `${{ secrets.GITHUB_TOKEN }}` (this is automatically populated by GitHub).

From what I've seen your personal token seems in correct format, but maybe it was automatically revoked to protect your account?

Hope this helps 👍

**Comment by qinyihao** [comment]
> Hello!
> 
> You should revoke your token now as you're not supposed to put them in plain text. Even if you blurred them on the image, anyone accessing your repository can actually see them 😅
> 
> It must be used through repository secrets (accessible through your repository settings) like said in this section : [Using GitHub action for metrics](https://github.com/lowlighter/metrics#%EF%B8%8F-using-github-action-on-your-profile-repository-5-min-setup).
> 
> There is a collapsible section "How to setup" which can be expanded and it contains a step-by-step tutorial with pictures 🙂
> 
> `committer_token` should litterally be set to `${{ secrets.GITHUB_TOKEN }}` (this is automatically populated by GitHub).
> 
> From what I've seen your personal token seems in correct format, but maybe it was automatically revoked to protect your account?
> 
> Hope this helps 👍

Thanks, Is `"token"` the right format? I would like to have a double check.

**Comment by qinyihao** [comment]
So I have to add these two to secret, right?
![image](https://user-images.githubusercontent.com/40647091/107122431-b68c4900-68d2-11eb-9595-bae6c88f370c.png)

**Comment by qinyihao** [comment, marked as answer]
Successfully built. Thank you!

**Comment by lowlighter** [comment]
Nice! Thanks for using metrics 😊
